docs/playbook/training-modules/Module-01.md

```diff
@@ -0,0 +1,90 @@
+We have created a `sandbox` department in the Data Platform, and an associated Google service account: `sandbox@dataplatform-stg.iam.gserviceaccount.com`
+- The Data Platform will use this account to connect to your Google sheet. 
+- Use the sharing settings of your sheet and grant “Viewer” permissions to this user.
+- [Detailed steps](https://playbook.hackney.gov.uk/Data-Platform-Playbook/playbook/ingesting-data/google-sheets-import#preparing-a-google-sheet-for-ingestion) are in the Playbook.
+
+
+### 3. Creating two ingestion jobs in Terraform
+Your Google sheet is going to be ingested by a job running in AWS [Glue](https://lbhackney-it.github.io/Data-Platform-Playbook/glossary#glue). You will write a bit of [Terraform](https://lbhackney-it.github.io/Data-Platform-Playbook/glossary/#terraform) that will deploy this job automatically in the AWS environment. You are going to write this code directly in your web browser in GitHub here, adding to the existing `Terraform script 26-google-sheets-imports.tf`.
+
+![Terraform script](../images/edit_terraform_google_sheets.png)
+
+You could also write the job locally within your interactve development environment, but this would remove the ability to test the code due to the way the Data Platform infrastructure is currently set up. Therefore, editing via the GitHub web interface will be easier in this case. 
+
+You will need to create two modules within the Terraform script as we are ingesting two datasets from the Google spreadsheet:
+- locations
+- vaccinations
+
+i. Go to the Playbook to view [detailed steps](https://playbook.hackney.gov.uk/Data-Platform-Playbook/playbook/ingesting-data/google-sheets-import#setting-up-the-aws-glue-job) on how to create a Terraform module.
+
+Additionally, please complete the following steps:
+
+ii. Your department name will be `sandbox` for the purpose of the training e.g. `module.department_sandbox`.
+
+iii. Suffix your new Terraform module name with the department `sandbox` and `your_name` so that each participant’s job will be easily identifiable. The module name will need to also reflect the individual dataset so make sure that the dataset name is also included within the module name e.g. `sandbox_joe_bloggs_covid_vaccinations`.
+
+iv. Prefix the dataset_name with your name e.g `joe_bloggs_covid_vaccinations`.
+
+v. No schedule is required as this is a one-off import, so this line does not need to be included within the module.
+
+
+### 4. Getting your jobs deployed to the Data Platform
+The next step is to commit your changes to the [Data Platform GitHub repository](https://github.com/LBHackney-IT/Data-Platform). Follow the instructions [here](https://playbook.hackney.gov.uk/Data-Platform-Playbook/playbook/getting-set-up/using-github#committing-your-changes-to-the-data-platform-project) on how to do this.
+
+> :warning: Please note that if you do not have access to the AWS sandbox account then this step will fail.
```

Comment:
Is this in the right place? I don't see why not having access to the AWS account would affect committing changes to github?

Reply:
have removed